### PR TITLE
Add lmd fork choice rule into chain module

### PR DESCRIPTION
**Pull request by wemeetagain**
Drive chain head with lmd ghost

### CODE REVIEW

File: src/chain/index.ts [hunk 1]
```diff
@@ -23,12 +25,14 @@ export class BeaconChain extends EventEmitter {
   private db: DB;
   private eth1: Eth1Notifier;
   private _latestBlock: BeaconBlock;
+  private forkChoice: LMDGHOST;
```
Comment (mpetrunic) [inline src/chain/index.ts:28]: Would be nice to set interface type here in case fork choice implementation changes

**Review by mpetrunic — CHANGES_REQUESTED** [review]

**Review by GregTheGreek — DISMISSED** [review]
WOOOOOP

File: src/chain/index.ts (hunk 1 above)
Comment (wemeetagain) [reply src/chain/index.ts:28]: :+1:  will make an issue for this

edit: will just include here

**Review by mpetrunic — DISMISSED** [review]

**Review by GregTheGreek — CHANGES_REQUESTED** [review]
CI test is failling

**Review by mpetrunic — APPROVED** [review]

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**